tests/unittests/graphTest.cpp

```diff
@@ -1068,3 +1068,16 @@ TEST(Graph, setType) {
   EXPECT_EQ(topK->getType(1), origTopKRes1);
   EXPECT_EQ(valRes1.getType(), origTopKRes1);
 }
+
+/// Check that verify doesn't allow for multiple writers to the same node.
+TEST(Graph, verifyOneWriter) {
+  Module M;
+  auto *F = M.createFunction("main");
+
+  auto *input = M.createPlaceholder(ElemKind::FloatTy, {5}, "input", false);
+  auto *output = M.createPlaceholder(ElemKind::FloatTy, {5}, "output", false);
+  F->createSave("Save1", input, output);
+  F->createSave("Save2", input, output);
+
+  EXPECT_DEATH(M.verify(), "");
```

Comment:
This check won't work in release mode.
The verifier does nothing in that mode (and that's sad, and I'd like we change that see https://github.com/pytorch/glow/issues/1517#issuecomment-415549954, but we're not there yet).

Reply:
Ah, true. Any suggestion on how to proceed here? We could wrap this test with `NDEBUG` I suppose...

Reply:
Sounds reasonable to me and unfortunately I don't have any other suggestions (aside what I said in #1517 :)).

Reply:
Done!